.distignore

```diff
@@ -31,6 +31,4 @@ node_modules
 *.tar.gz
 *.zip
 sync.sh
-package-lock.json
-.github
-.wordpress-org
```

Comment:
If these lines are mistakenly deleted from here, revert it

Reply:
No actually I added them in the first place, but once checked the original files and they were not there in original files I removed them.